### PR TITLE
Feature/simple facets

### PR DESCRIPTION
**Pull request by djtfmartin**
Bug fixes and minor UI layout improvements to make specieslists looks more like other apps (biocache)

This deployed to lists-test.ala.org.au for review

### CODE REVIEW

File: grails-app/i18n/messages_ca.properties [hunk 1]
```diff
@@ -103,4 +103,5 @@ upload.instructions.hasList=Atenció\: tornar a enviar una llista existent farà
 
 facet.guid\:null=Tàxons no reconeguts
 
-error.message.reloadListPermission=No teniu permisos per tornar a carregar {0}.
\ No newline at end of file
+error.message.reloadListPermission=No teniu permisos per tornar a carregar {0}.
+australian.iconic.species=Australian iconic species
```
Comment (nickdos) [inline grails-app/i18n/messages_ca.properties]: This i18n code has been added to all the language files as an english "value". I think the EN value should only be in the first (EN) file and not all the others (i18n falls back to EN when missing value anyway).

File: grails-app/assets/stylesheets/AlaBsAdditions.css [hunk 2]
```diff
@@ -101,29 +101,36 @@ div#download {
...
-    padding-left:15px;
-}
-
-#currentFilters > ul > li {
-    margin: 0px;
-    padding: 0px;
-    text-indent: -1.3em;
-    margin-left: 0;
-}
+/*#currentFilters > ul {*/
```
Comment (nickdos) [inline grails-app/assets/stylesheets/AlaBsAdditions.css]: delete ?

File: grails-app/i18n/messages_ca.properties (hunk 1 above)
Comment (djtfmartin) [reply grails-app/i18n/messages_ca.properties]: removed

File: grails-app/assets/stylesheets/AlaBsAdditions.css (hunk 2 above)
Comment (djtfmartin) [reply grails-app/assets/stylesheets/AlaBsAdditions.css]: removed

**Comment by nickdos** [comment]
All good to merge...

**Pull request merged**